### PR TITLE
Team stats js

### PR DESCRIPTION
**Pull request by jadekstewart3**
Add team_info and best_season methods with helper methods and passing tests

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -341,5 +341,135 @@ def team_goals_by_season(season_id)
...
+   best = pair_season_with_results_by_team(team_id)
+ 
+   best.each do |season, results|
+     best_season_hash[season] = results.count("WIN") / results.count.to_f
+   end
+   best_season_for_team = best_season_hash.max_by {|k,v| v}
+   best_season_for_team[0]
+ end
+
+ def worst_season(team_id)
```
Comment (KaraJoHo) [inline lib/stat_tracker.rb:465]: niiiiiiice

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -341,5 +341,135 @@ def team_goals_by_season(season_id)
...
+    best = pair_season_with_results_by_team(team_id)
+   
+    best.each do |season, results| 
+      best_season_hash[season] = results.count("WIN") / results.count.to_f 
+    end
+    best_season_for_team = best_season_hash.max_by {|k,v| v}
+    best_season_for_team[0]
+  end
+
+	def team_info(team_id)
```
Comment (KaraJoHo) [inline lib/stat_tracker.rb:405]: this is sweet! I like the use of iteration to make this

**Review by KaraJoHo — APPROVED** [review]

File: lib/stat_tracker.rb (hunk 2 above)
Comment (sgwalker327) [reply lib/stat_tracker.rb:405]: Nicely done!

File: lib/stat_tracker.rb (hunk 1 above)
Comment (sgwalker327) [reply lib/stat_tracker.rb:465]: heck yeah

**Review by sgwalker327 — COMMENTED** [review]
Well done!

**Pull request merged**